deploymentscripts/deploy-pi.sh

```diff
@@ -23,8 +23,8 @@ echo 'eval "$(rbenv init -)"' >> ~/.bashrc
 export PATH="$HOME/.rbenv/bin:$PATH"
 eval "$(rbenv init -)"
 git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
-CONFIGURE_OPTS="--disable-install-doc" rbenv install 2.6.5 --verbose
-rbenv global 2.6.5
+CONFIGURE_OPTS="--disable-install-doc" rbenv install 2.7.2 --verbose
+rbenv global 2.7.2
 gem install bundler -v 1.16.1
```

Comment:
bump version to 1.17.2 here as well.

Reply:
Bumped and added `--no-document` to match `ubuntu-prep.sh`.